.github/workflows/ci.yml

```diff
@@ -87,6 +87,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+      number: [1,2,3,4,5,6,7,8,9]
```

Comment:
I know these were added to 'stress' test the integration tests, but I'm going to 'request changes to make sure we don't accidentally merge this change.

Reply:
Apologies, I forgot to remove that before committing 😅 . Just fixed it.